source/SyntheticTopology/SierpinskiAxioms.lagda

```diff
@@ -0,0 +1,124 @@
+---
+title:        Axioms about the Sierpinski object
+authors:      [Martin Trucchi , Ayberk Tosun]
+date-started: 2024-05-28
+---
+
+\begin{code}
+
+{-# OPTIONS --safe --without-K #-}
+
+open import MLTT.Spartan
+open import UF.Base
+open import UF.FunExt
+open import UF.PropTrunc
+open import UF.Sets
+open import UF.Sets-Properties
+open import UF.Subsingletons
+open import UF.SubtypeClassifier
+open import SyntheticTopology.SierpinskiObject 
+
+module SyntheticTopology.SierpinskiAxioms
+        (𝓤  𝓥 : Universe)
+        (fe : Fun-Ext)
+        (pe : Prop-Ext)
+        (pt : propositional-truncations-exist)
+        (𝕊 : Generalized-Sierpinski-Object fe pe pt 𝓤 𝓥)
+        where
+
+open import UF.Logic
+
+open AllCombinators pt fe
+open Sierpinski-notations fe pe pt 𝕊
+
+
+\end{code}
+
+We write down the Dominance Axiom and Phoa’s Principle.
+
+First, the Dominance Axiom:
+
+\begin{code}
+
+contains-top : Ω 𝓥
+contains-top = is-open-proposition ⊤
+
+openness-is-transitive : Ω ((𝓤 ⁺) ⊔ 𝓥)
+openness-is-transitive =
+ Ɐ u ꞉ (Ω 𝓤) ,
+  (is-open-proposition u
+   ⇒ (Ɐ p ꞉ (Ω 𝓤) ,
+    (u ⇒ (is-open-proposition p))
+     ⇒ (is-open-proposition (u ∧ p))))
+
+is-synthetic-dominance : Ω (𝓤 ⁺ ⊔ 𝓥)
+is-synthetic-dominance = contains-top ∧ openness-is-transitive
+
+\end{code}
+
+Phoa’s Principle:
```

Comment:
It might be a good idea to divide the file up into sections using `\section{...}`. But once again, this is not that important at this point.

Reply:
Done ? Not used any submodules tho